eng/config/globalconfigs/ApiShim.globalconfig

```diff
@@ -1,10 +1,15 @@
-is_global = true
-
```

Comment:
This should be kept

Reply:
So what I was trying to deal with here was that without this change [I get the following error](https://dev.azure.com/dnceng/public/_build/results?buildId=988457&view=logs&j=1f132584-ab08-5fbf-e5d0-5685de454342&t=76c675cd-a096-5dd5-8431-7aa49ff610be&l=151):
```
CSC : error MultipleGlobalAnalyzerKeys: Multiple global analyzer config files set the same key 'dotnet_diagnostic.ca2227.severity' in section 'Global Section'. It has been unset. Key was set by the following files:
```
because it's set in both `ApiShim.globalconfig` and `NonShipping.globalconfig`. Basically it seems that specifically for globalconfig files there's no good way to have a "hierarchy" so that the settings in ApiShim.globalconfig override thos of NonShipping.globalconfig (when they overlap). Is there maybe something I missed in the spec which solves this? Like an "importance" setting?

Reply:
Let's leave this part of the change out for now and I'll split these files so we have a better way to separate them.

Reply:
Ah, easy. We can do this...

Reply:
Start with https://github.com/dotnet/aspnetcore-tooling/pull/3121#discussion_r574134555, and then you may need to define additional exclusions since NonShipping.globalconfig will no longer apply in these projects.